packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -170,18 +170,54 @@ final togglePlatformMode = ServiceExtensionDescription<String>(
   gaItem: ga.togglePlatform,
 );
 
-final toggleSelectWidgetMode = ToggleableServiceExtensionDescription<bool>._(
+// Legacy extension to show the inspector and enable inspector select mode.
+final toggleOnDeviceWidgetInspector =
+    ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.inspector.show',
+  // Technically this enables the on device widget inspector but for older
+  // versions of package:flutter it makes sense to describe this extension as
+  // toggling widget select mode as it is the only way to toggle that mode.
   description: 'Select Widget Mode',
   icon: FlutterIcons.locate,
   enabledValue: true,
   disabledValue: false,
-  enabledTooltip: 'Disable Select Widget Mode',
-  disabledTooltip: 'Enable Select Widget Mode',
+  enabledTooltip: 'Disable select widget mode',
+  disabledTooltip: 'Enable select widget mode',
```

Comment:
should these tooltips match the new ones with exit / enter instead of disable / enable?

Reply:
These tooltips should remain unchanged as they reflect the legacy use case which still works as it did before.